compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1326,42 +1303,32 @@ LogicalResult AttentionOp::verify() {
         shape[pos] = valShape[i];
       }
       if (shape[pos] != valShape[i]) {
-        return op->emitError("Shape Mismatch for ")
+        return attnOp->emitError("Shape Mismatch for ")
                << operandName << ". Expected: " << shape[pos]
                << " Got: " << valShape[i];
       }
     }
     return success();
   };
 
-  if (failed(checkShape("Query", getQueryType().getShape(), getQueryMap())) ||
-      failed(checkShape("Key", getKeyType().getShape(), getKeyMap())) ||
-      failed(checkShape("Value", getValueType().getShape(), getValueMap()))) {
+  if (failed(checkShape("Query", getQuery().getType().getShape(),
+                        getQueryMap())) ||
+      failed(checkShape("Key", getKey().getType().getShape(), getKeyMap())) ||
+      failed(checkShape("Value", getValue().getType().getShape(),
+                        getValueMap())) ||
```

Comment:
The change looks unnecessary, and the `get*Type()` looks better to me. Why do you change them? Can you revert the change?

Reply:
SG will add it back. This was more for prototyping purpose as we change the way we initialize the DpsInputs i.e from SmallVector<ShapedType> to a bunch of shapedType. But no reason we can't have same helper functions. :)

Reply:
done!